toot/tui/utils.py

```diff
@@ -51,11 +51,18 @@ def _gen():
     return list(_gen())
 
 
-def highlight_hashtags(line, attr="hashtag"):
-    return [
-        (attr, p) if p.startswith("#") else p
-        for p in re.split(HASHTAG_PATTERN, line)
-    ]
+def highlight_hashtags(line, followed_tags, attr="hashtag",\
```

Comment:
You don't need a backspace for content inside parenthesis. I mostly avoid using backslashes.